packages/remix/src/ssr/rootAuthLoader.ts

```diff
@@ -25,8 +25,8 @@ interface RootAuthLoader {
 
 export const rootAuthLoader: RootAuthLoader = async (
   args: LoaderFunctionArgs,
-  handlerOrOptions: any,
-  options?: any,
+  handlerOrOptions?: RootAuthLoaderCallback<RootAuthLoaderOptions> | RootAuthLoaderOptions,
+  options?: RootAuthLoaderOptions,
```

Comment:
❓ since `RootAuthLoader` type is defined above to have `callback: RootAuthLoaderCallback<RootAuthLoaderOptions> ` and `options?: RootAuthLoaderOptions`, do you think this change is breaking change?

Reply:
I think this only affects the types inside the function. (Due to more strict types set to `rootAuthLoader: RootAuthLoader`) But I am not 100% sure.
A safer approach would be to type cast with `as` those variables inside the function, and the signature will not be affected for sure

Reply:
Would you think this is a cleaner approach? https://github.com/clerkinc/javascript/blob/c7e61d9eb067694b790b52983cd57f968938ed69/packages/nextjs/src/server/withClerkMiddleware.ts#L44

cc @dimkl , @panteliselef

Reply:
yes

Reply:
me too. but i will revert this change for the moment to merge this PR and we will refactor it in the future